onert-micro/luci-interpreter/include/luci_interpreter/core/ParamsType.h

```diff
@@ -52,6 +52,15 @@ enum class MirrorPadMode
   SYMMETRIC,
 };
 
+enum class OperationGraphStatus
+{
+  USUAL,
+
+  START,
+  MIDDLE,
+  END,
+};
+
```

Comment:
Introduce three types of the operations:
USUAL - it is  all operations by default;
START - it is operation that is first in the chain of intermediate quantized tensors (input of the operation is FLOAT type, output is INT16 type)
MIDDLE - for operations in the middle of the chain of intermediate quantized tensors (input and ouput is INT16)
END - output of the chain of intermediate quantized tensors (input type is INT16, output is FLOAT)